src/Logger.php

```diff
@@ -185,9 +183,8 @@ public function log($level, $message, array $context = [])
             if ($this->traceLevel > 0) {
                 $count = 0;
                 $ts = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
-                array_pop($ts); // remove the last trace since it would be the entry script, not very useful
```

Comment:
Why was it removed?

Reply:
Because since this can be used outside Yii, we cannot make assumptions about whether there's an empty script or not.

Example: test.php contains:

```php
<?php

require 'vendor/autoload.php';

$logger = (new \Yiisoft\Log\Logger())
    ->setTraceLevel(3);

$monolog = new \Monolog\Logger('my-logger');
$monolog->pushHandler(new \Monolog\Handler\RotatingFileHandler('mono.log'));

$psrSyslog = (new \Yiisoft\Log\PsrTarget($monolog))
    ->setLogVars(['_SERVER']);

$logger->addTarget($psrSyslog);
$logger->debug('This is a debug message');
```

Last entry in backtrace is `test.php` which is definitely something we want to see.